docker-compose.yml

```diff
@@ -35,6 +35,7 @@ services:
       - 3334:3334
     volumes:
       - ../v3-website:/usr/src/app
+      - ./cache:/usr/src/cache
```

Comment:
I originally thought about using a specific Docker volume for the cache, but we can discuss this later